src/AbstractCommand.php

```diff
@@ -31,6 +31,14 @@ public function options()
     /**
      * @inheritDoc
      */
+    public function option($name)
```

Comment:
Not sure why you need this... once you pass `$this->options()` than all required stuff is known to be defined.

If anything, this points to the need for #6